File: movieApp/app/src/main/java/com/lbc/practice/movieapp/adapter/PlayingListContract.java
package com.lbc.practice.movieapp.adapter;

import com.lbc.practice.movieapp.callback.OnItemClickListener;
import com.lbc.practice.movieapp.model.PlayingMovieResult;

import java.util.List;

public interface PlayingListContract {

    interface View {
        void notifyAdapter();
        void setOnclickListener(OnItemClickListener onclickListener);
    }

    interface PlayingModel {
        void addMovieInfo (List<PlayingMovieResult.ResultsBean> enrollIntroduces);
        void clearItem();
    }
}

File: movieApp/app/src/main/java/com/lbc/practice/movieapp/adapter/PlayingListContract.kt
package com.lbc.practice.movieapp.adapter;

import com.lbc.practice.movieapp.callback.OnItemClickListener;
import com.lbc.practice.movieapp.data.PlayingMovieResult;



interface PlayingListContract {

    interface View {
        fun notifyAdapter()
        fun setOnclickListener(onclickListener: OnItemClickListener)
    }

    interface PlayingModel {
        fun addMovieInfo(enrollIntroduces: kotlin.collections.MutableList<PlayingMovieResult.ResultsBean>)
        fun clearItem()
    }
}